test/model.test.js

```diff
@@ -1075,105 +1031,11 @@ describe('Model', function() {
         const doc = await post.save();
         assert.ok(doc);
       });
-
-      it('async', async function() {
```

Comment:
No, it was removed because we don't support callback-based async validators anymore as of Mongoose 6. It looks like this particular syntax still works, but we don't need tests for it anymore.